app/views/users/show.html.erb

```diff
@@ -1,2 +1,22 @@
-<h1>Users#show</h1>
-<p>I am in app/views/users/show.html.erb</p>
\ No newline at end of file
+<div class=''>
+    <%= render partial: 'partials/user_profile', locals: { user: @user } %>
+    <div class='mb-4'>
+        <h3 class='text-xl bp-3'>Bio <h3>
+        <div class='text-gray-500 border-b-2 border-gray-200 pb-3'>
+            <%=@user.bio%>
+        </div>
+    </div>
+    <div class='mb-4'>
+        <h3 class='text-xl bp-4 text-gray-400'>Posts<h3>
+        <ul class='flex flex-col gap-2'>
+            <% @user.three_most_recent_posts.each_with_index do |post, index| %>
+                <li>
+                    <%= render partial: 'partials/post_item', locals: { post: post, index:index } %>
+                </li>
+            <% end %>
+        </ul>
+    </div>
+    <div>
+        <%= link_to 'See all posts', user_posts_path(@user.id), method: :get, class: 'bg-gray-700 text-white py-2 px-6 rounded'%>
```

Comment:
- [ ] When clicking on this button an error arose, please 🙏🏽 fix it to be able to see all posts 👇🏽 
![image](https://github.com/Rafiduzzaman/blog-app/assets/58642949/e1d7814e-4d18-4999-b6d1-5908b1f4da22)